src/ResXManager.View/Visuals/TranslationItem.cs

```diff
@@ -61,8 +61,14 @@ public bool Apply(string? valuePrefix, string? commentPrefix)
             if (!_entry.CanEdit(_entry.NeutralLanguage.CultureKey))
                 return false;
 
-            _entry.Comments.SetValue(_entry.NeutralLanguage.CultureKey, commentPrefix + existingComment);
-
+            foreach (var languageKey in _entry.Languages)

```

Comment:
You apply the existing neutral comment to any language, overriding any existing language specific comment; also you add the prefix to any language - is this really intended?
Shouldn't this rather modify only the comment of the target language?

What about backward compatibility? The neutral comment is now no longer updated, what about users that rely on this?

Reply:
Yes, it looks like the target language was the intent here as opposed to going over each language. 

I hadn't really considered backwards compatibility because this seemed like the more correct implementation of the original pull request that added the option, but technically you're correct. I suppose the best option would be to add a setting for the original behavior, and the new behavior?

Reply:
I've been wrestling with the best way to handle backwards compatibility for this case (trying to keep in mind the best user experience as well as the best/cleanest code implementation). Here is a proposed alternative solution #550